### PR TITLE
BEGIN THE REAPING

### PR DESCRIPTION
**Pull request by ryanpbrewster**
i.e., drop libraryId, visibility, and organizationId from keep

So far, only organizationId is gone

@leogrim

### CODE REVIEW

File: kifi-backend/modules/search/app/com/keepit/search/index/graph/keep/KeepIndexable.scala [hunk 1]
```diff
@@ -115,12 +115,14 @@ case class KeepIndexable(keep: Keep, sourceAttribution: Option[SourceAttribution
       }
     }
 
+    /* TODO(ryan): figure out the correct thing to do here
     keep.organizationId.foreach { orgId =>
       doc.add(buildKeywordField(orgField, orgId.id.toString))
       if (keep.visibility == LibraryVisibility.PUBLISHED || keep.visibility == LibraryVisibility.ORGANIZATION) {
         doc.add(buildKeywordField(orgDiscoverableField, orgId.id.toString))
       }
     }
```
Comment (ryanpbrewster) [inline kifi-backend/modules/search/app/com/keepit/search/index/graph/keep/KeepIndexable.scala:126]: @leogrim

Comment (ryanpbrewster) [inline kifi-backend/modules/search/app/com/keepit/search/index/graph/keep/KeepIndexable.scala:126]: I assume I want to change `KeepIndexable` to contain a `CrossServiceKeep` and change `CrossServiceKeep` from having `libraries: Set[Id[Library]]` to having `libraries: Set[CrossServiceKeep.Library]` where `case class Library(id: Id[Library], visibility: LibraryVisibility, organizationId: Option[Id[Organization]])` or something

**Comment by andrewconner** [comment]
:fist:

File: kifi-backend/modules/search/app/com/keepit/search/index/graph/keep/KeepIndexable.scala (hunk 1 above)
Comment (leogrim) [inline kifi-backend/modules/search/app/com/keepit/search/index/graph/keep/KeepIndexable.scala:126]: Sounds good!

**Pull request closed without merging**